packages/docusaurus/src/webpack/__tests__/__snapshots__/base.test.ts.snap

```diff
@@ -23,21 +23,23 @@ Object {
   "@docusaurus/useIsBrowser": "../../../../client/exports/useIsBrowser.ts",
   "@generated": "../../../../../../..",
   "@site": "",
-  "@theme-init/PluginThemeComponentOverridden": "pluginThemeFolder/PluginThemeComponentOverridden.js",
```

Comment:
`PluginThemeComponentOverriddenByUser` is no longer getting a `@theme-init` alias because it's not provided by two themes, only by the user and one theme.